Solutions/MailGuard 365/Data Connectors/MailGuard365.json

```diff
@@ -0,0 +1,108 @@
+{
+    "id": "MailGuard365",
+    "title": "MailGuard 365",
+    "publisher": "MailGuard365",
+    "descriptionMarkdown": "MailGuard 365 Enhanced Email Security for Microsoft 365. Exclusive to the Microsoft marketplace, MailGuard 365 is integrated with Microsoft 365 security (incl. Defender) for enhanced protection against advanced email threats like phishing, ransomware and sophisticated BEC attacks.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "MailGuard365_Threats_CL",
+            "baseQuery": "MailGuard365_Threats_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "All phishing threats stopped by MailGuard 365",
+            "query": "MailGuard365_Threats_CL \n | where where Category == \"Phishing\""
```

Comment:
Please remove the extra "where" form the query, this should fix the failing validation.

Reply:
Please do fix the ARM TTK validation as well.

Reply:
Thank you, I've fixed the KQL validation and the ARM TTK validation as well.